File: 백준/Bronze/2588. 곱셈/곱셈.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(), b = in.nextInt();
        int res = a*b;
        while(b > 0){
            System.out.println(a*(b%10));
            b = (b-b%10)/10;
        }
        System.out.print(res);
    }
}

File: 백준/Bronze/2588. 곱셈/곱셈.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var a = sc.nextInt(); var b = sc.nextInt()
    var res : Int = a*b
    while(b > 0){
        println(a * (b%10))
        b = (b-b%10)/10
    }
    print(res)
}